WcaOnRails/app/views/competitions/edit.html.erb

```diff
@@ -40,26 +41,26 @@
         </div>
       </div>
     <% else %>
-      <% if @competition.isConfirmed? && @competition.showAtAll? %>
+      <% if is_actually_confirmed && @competition.showAtAll? %>
```

Comment:
Oh, I get it now. I was under the impression that `@competition == Competition.find(@competition.id)`.

I wouldn't test it either if it's just some usability thing. But I think one simple feature test for editing a competition might be worth writing in order to catch stupid type errors.